lib/components/chat_history/twitch/shoutout_event.dart

```diff
@@ -16,12 +21,50 @@ class TwitchShoutoutEventWidget extends StatelessWidget {
                   text: "Shoutout was given to ",
```

Comment:
Can we match the Twitch text for shoutouts? @SputNikPlop do you know what it is?

Reply:
> Can we match the Twitch text for shoutouts? @SputNikPlop do you know what it is?

It looks something like this
![Screen Shot 2023-01-27 at 4 16 43 PM](https://user-images.githubusercontent.com/100245448/215229094-77dc3492-5d11-4394-a4ae-1302ff127b9e.png)